react/components/LogoutCustomerSession.js

```diff
@@ -14,50 +14,73 @@ import TelemarketingIcon from '../icons/TelemarketingIcon'
 
 /** Component that shows the client info calls the setSession function  to logout. */
 export default class LogoutCustomerSession extends Component {
+  
   handleHeaderRendering = () => {
-    const { client } = this.props
+    const { client, mobile } = this.props
 
     return (
-      <div className="flex align-center">
-        <CustomerIcon />
-        <div className="pa2">
-          {client.email ? `${truncateString(client.email, 25)}` : null}
+      mobile ? (
+        <div className="flex align-center">
+          <CustomerIcon />
+          <div className="pa2">
+            {this.getClientName()}
+          </div>
         </div>
-      </div>
+      ) : (
+        <div className="flex align-center">
+          <CustomerIcon />
+          <div className="pa2">
+            {client.email ? `${truncateString(client.email, 25)}` : null}
+          </div>
+        </div>
+      )
     )
   }
 
+  getClientName = () => {
+    const { client } = this.props
+
+    return client.name.includes('null') ? client.email.slice(0, client.email.indexOf('@')) : client.name
+  }
+
   render() {
-    const { intl, client, loading, onDepersonify, attendantEmail } = this.props
+    const { intl, client, loading, onDepersonify, attendantEmail, mobile } = this.props
 
     return (
-      <div className="vtex-telemarketing__logout">
+      <div className={`textPositionStyle ${mobile && 'w-50'}`}>
```

Comment:
```suggestion
      <div className="vtex-telemarketing__logout w-50-s">
```
Don't remove the customization tokens, they will be used to provide further customization of the component.
Where was textPositionStyle coming from?
To avoid using mobile variable, use media queries.